plugins/com.google.cloud.tools.eclipse.appengine.libraries/schema/com.google.cloud.tools.eclipse.appengine.libraries.exsd

```diff
@@ -0,0 +1,243 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.google.cloud.tools.eclipse.appengine.libraries" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="com.google.cloud.tools.eclipse.appengine.libraries" id="com.google.cloud.tools.eclipse.appengine.libraries" name="App Engine Libraries"/>
+      </appinfo>
+      <documentation>
+         Provides an extension point to define libraries that will be resolved using the Maven coordinates. These libraries then can be added to App Engine projects.
```

Comment:
Probably should add that this is intended for internal use only and may change without notice.